src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -1036,14 +1036,14 @@ private void createMultipleSlackBusesEquations(EquationSystem<AcVariableType, Ac
             LfBus firstSlackBus = slackBuses.get(0);
             for (int i = 1; i < slackBuses.size(); i++) {
                 LfBus slackBus = slackBuses.get(i);
-                // example for 2 slack buses
-                // 0 = slack_p1 - slack_p2
-                // 0 = slack_p1 - slack_p3
+                // example for 3 slack buses
+                // target_p2 = slack_p2 - slack_p1 + inj_p2
+                // target_p3 = slack_p3 - slack_p1 + inj_p2
```

Comment:
Not sure about my remark but shouldn't we rather have : 
```
// target_p2 = slack_p2 - slack_p1
// target_p3 = slack_p3 - slack_p1
```

Reply:
you are absolutely right ... thanks / fixed